test/trivial/diten/testIntExp4.good

```diff
@@ -1 +1 @@
-error: 0 cannot be raised to a negative power
```

Comment:
I did not see anything in your compiler changes above that would cause this change in the output. Could you please double-check that you git-added everything that needs to be? Ditto the changes to type-tests.isSubtype.good below.